commands/reverseintake.py

```diff
@@ -0,0 +1,24 @@
+from commands2 import CommandBase
+
+from subsystems.intakesubsystem import IntakeSubsystem
+
+
+class ReverseIntake(CommandBase):
+    def __init__(self, intake: IntakeSubsystem) -> None:
+        CommandBase.__init__(self)
+        self.setName(__class__.__name__)
+        self.intake = intake
+        self.addRequirements(intake)
+
+    def initialize(self) -> None:
+        print(f"Command: {self.getName()}")
+        self.intake.reverseIntake()
+
+        if self.intake.isIntakeDeployed():
+            self.intake.runIntake(self.intake.intakeReversed)
```

Comment:
I noted this below - it is generally some iffy practice to take values directly from classes, instead these should be handled within the subsystem itself and you should be calling onto those values